docs/template_plugin/tests/functional/op_reference/ti.cpp

```diff
@@ -0,0 +1,66 @@
+// Copyright (C) 2018-2021 Intel Corporation
```

Comment:
Why not tensor_iterator.cpp?

Reply:
renamed. Actually there are some tests in other places with 'ti.cpp' name for TensorIterator tests